erpnext/accounts/doctype/account/account.json

```diff
@@ -253,4 +253,4 @@
  "sort_order": "ASC",
  "states": [],
  "track_changes": 1
```

Comment:
Modified date has not updated, due to which this changes won't be reflect on the new site after the migrate